src/shared/components/palette/utils/first-error.pipe.spec.ts

```diff
@@ -1,74 +1,37 @@
-import { FirstErrorPipe } from './first-error.pipe';
```

Comment:
This makes absolutely no sense! This is `.spec.ts` file yet you've somehow turned it into a pipe and it's now a copy of the (wrong) `FirstErrorPipe`!